styles/Header.module.css

```diff
@@ -6,6 +6,7 @@
     flex-wrap: wrap;
     height: 75px;
     padding: 0 50px;
+    top: 0;
```

Comment:
This is to fix the header, it won't be sticky without it. Deleted it accidentally at commit [e4bbb718063849ad27fb73b871f456ad5ebc3cc5](https://github.com/LRode/halfmoon-frontend/commit/e4bbb718063849ad27fb73b871f456ad5ebc3cc5)